great_expectations/expectations/metrics/util.py

```diff
@@ -608,26 +615,44 @@ def column_reflection_fallback(
             # noinspection PyUnresolvedReferences
             if dialect.name.lower() == GXSqlDialect.REDSHIFT:
                 # Redshift needs temp tables to be declared as text
-                query: Select = (
+                query = (
                     sa.select([sa.text("*")]).select_from(sa.text(selectable)).limit(1)
                 )
             else:
-                query: Select = (
-                    sa.select([sa.text("*")]).select_from(selectable).limit(1)
-                )
+                query = sa.select([sa.text("*")]).select_from(selectable).limit(1)
         result_object = sqlalchemy_engine.execute(query)
         # noinspection PyProtectedMember
         col_names: List[str] = result_object._metadata.keys
         col_info_dict_list = [{"name": col_name} for col_name in col_names]
     return col_info_dict_list
 
 
+@overload
+def get_dbms_compatible_column_names(
```

Comment:
❤️ Overloaded method signatures in `Python`?!  @Kilo59 this is amazing!!!

Reply:
🎊  https://mypy.readthedocs.io/en/stable/more_types.html?highlight=overload#function-overloading